src/shared/api/entities/schemas.ts

```diff
@@ -48,3 +48,9 @@ export type UserWithTokenResponse = {
   accessToken: string;
   refreshToken: string;
 };
+
+export type CompanyListResponse = {
+  id: string;
+  name: string;
+  description: string;
+};
```

Comment:
해당 DTO는 리스트가 아니니 CompanyResponse가 더 적합한 네이밍입니다. 그리고 API 명세 반영이 안 된 거 같네요? 확인해주시면 감사하겠습니다.
```suggestion
export type CompanyResponse = {
  id: string;
  name: string;
  description: string;
};
```